weekly-scan/master-controller.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: weekly-scan-master-controller
+labels:
+  type: "controller-job"
+objects:
+  - kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: weekly-scan-master-controller-script
+    data:
+      script: |-
+        GOT_ANY='0'
+        OS_TOKEN=`cat /opt/jenkins-secret/token`
+        oc login ${OS_MASTER} --insecure-skip-tls-verify=true --token=${OS_TOKEN};
+        for item in `oc get bc -o name --selector type=weekly-scan`; do
+            GOT_ANY='1'
+            oc start-build ${item};
+            sleep ${BATCH_DELAY}
+        done
+        if [ ${GOT_ANY} == '0' ]; then
```

Comment:
This is to ensure there is proper messaging only. If no jobs got filtered, then the logs just show it. There is no other purpose. It will work fine without the logic as well.

Agreed the chances of this happening are less then 1 percent. But it is still a chance, and i rather handle it now then have to deal with it later.